thirdeye-ui/src/app/components/visualizations/time-series-chart/chart-core/chart-core.component.tsx

```diff
@@ -59,14 +59,16 @@ export const ChartCore: FunctionComponent<ChartCoreProps> = ({
     onMouseMove,
     onMouseEnter,
 }) => {
+    // `allSeries` will contain data with `enabled = false` as well
+    // Filter it out in the beginning so the rest of the component
+    // only has to care about interacting with the `enabled = true` data
+    const series = allSeries.filter((s) => s.enabled);
```

Comment:
Rename `series` => `enabledTimeSeries` to be explicit

Reply:
Renamed